src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {Text, TouchableOpacity, View} from 'react-native';
+import Tooltip from 'react-native-walkthrough-tooltip';
+import MemoIcQuestionMark from '../../../assets/icons/Ic_question_mark';
+import {normalize} from '../../../utils/fonts';
+import {LockIcon, PencilIcon} from '../../../assets';
+import {COLORS} from '../../../utils/theme';
+
+export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
+  const [isTooltipShown, setIsTooltipShown] = React.useState(false);
+
+  return (
+    <View
+      style={{
+        backgroundColor: COLORS.gray100,
+        borderRadius: 12,
+        paddingVertical: 8,
+        paddingHorizontal: 4
+      }}>
+      <View
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          alignItems: 'center'
+        }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, these inline styles should be moved to a StyleSheet object.

```diff
- style={{
-   display: 'flex',
-   flexDirection: 'row',
-   justifyContent: 'space-between',
-   alignItems: 'center'
- }}
+ style={styles.row}
```
And add this to your StyleSheet object:
```javascript
const styles = StyleSheet.create({
  row: {
    display: 'flex',
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center'
  },
  // other styles...
});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->